community/modules/compute/gke-node-pool/variables.tf

```diff
@@ -24,6 +24,12 @@ variable "cluster_id" {
   type        = string
 }
 
+variable "zones" {
+  description = "A list of zones to be used. Zones must be in region of cluster. If null, cluster zones will be picked. Note `zones` not `zone`; does not work with `zone` deployment variable."
```

Comment:
It would be good to document the default behavior. What happens if you do not specify zones?